### PR TITLE
Fix misleading error message.

### PR DESCRIPTION
**Pull request by dvalinrh**
An error message indicates fio test.  This is misleading as this tool can be used by many different tests.

### CODE REVIEW

File: grab_disks [hunk 1]
```diff
@@ -89,7 +89,7 @@ fi
 # to make sure the disk exist.
 #
 if [ $numb_disks -eq 0 ]; then
-	echo  "Need to have disks to do fio"
+	echo  "Need to have disks to perform desks, non provided."
```
Comment (frival) [inline grab_disks:92]: Um ... I'm just gonna ask.  "desks" and "non" ?

**Review by frival — COMMENTED** [review]
Entertaining typos.

File: grab_disks [hunk 2]
```diff
@@ -89,7 +89,7 @@ fi
 # to make sure the disk exist.
 #
 if [ $numb_disks -eq 0 ]; then
-	echo  "Need to have disks to do fio"
+	echo  "Need to have disks to perform disks, none provided."
```
Comment (kdvalin) [inline grab_disks:92]: > Need to have disks to perform disks

Yes this floor is made of floor. 

Maybe something like
```suggestion
	echo  "Need to have disks to perform test, none provided."
```

**Review by kdvalin — CHANGES_REQUESTED** [review]

**Review by frival — APPROVED** [review]
LGTM

**Review by kdvalin — APPROVED** [review]
LGTM
